context/app/static/js/pages/search/DevSearch.jsx

```diff
@@ -22,12 +23,15 @@ function DevSearch() {
     // Search results field which will be appended to detailsUrlPrefix:
     idField: 'uuid',
     // Search results fields to display in table:
-    resultFields: [
-      field('entity_type', 'type'),
-      field('display_doi', 'ID'),
-      field('mapped_last_modified_timestamp', 'Last Modified'),
-      field('mapper_metadata.size', 'Doc Size'),
-    ],
+    resultFields: {
+      table: [
+        field('entity_type', 'type'),
+        field('display_doi', 'ID'),
+        field('mapped_last_modified_timestamp', 'Last Modified'),
+        field('mapper_metadata.size', 'Doc Size'),
+      ],
+      tile: [],
```

Comment:
Ditto from the earlier PR, wonder if we can just keep this flatter and request a superset.

Reply:
We agreed this is fine for now, more in #1303.